### PR TITLE
Add 2020-10-10 minutes

### PR DESCRIPTION
**Pull request by wltan**
Added meeting minutes for 10 October.

### CODE REVIEW

File: docs/minutes/20201010.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+Issue: Need to support UI-dependent behaviour for commands.
+
+Solution: Pass the UI state to the command parser.
+
+### Actionables
+
+Wei Liang:
+- Create {Add, Delete} × {Expense, Income} Command
+  - Generic AddCommand that works based on the current UiState
```
Comment (ianyong) [inline docs/minutes/20201010.md:18]: Possible to add backticks to parts which reference code? i.e. `AddCommand`

**Review by ianyong — CHANGES_REQUESTED** [review]

File: docs/minutes/20201010.md (hunk 1 above)
Comment (wltan) [reply docs/minutes/20201010.md]: Fixed

**Review by yongping827 — APPROVED** [review]
LGTM

**Review by ianyong — APPROVED** [review]
LGTM!

**Pull request merged**